### PR TITLE
Update for the french translation

### PR DESCRIPTION
**Pull request by sivolc73**
Translated the new text and adjusted some previous translations now that I used the app a fair bit.

Is tip2 even working? or is this broken? (double click on server to open settings on line 30)

### CODE REVIEW

File: lib/l10n/app_fr.arb [hunk 1]
```diff
@@ -53,33 +53,33 @@
...
   "server": "Serveur",
   "device": "Appareil",
   "event": "Évènement",
   "duration": "Durée",
   "priority": "Priorité",
-  "next": "Next",
-  "previous": "Previous",
+  "next": "Suivant",
+  "previous": "Précédent",
   "lastImageUpdate": "Last Image Update",
```
Comment (bdlukaa) [inline lib/l10n/app_fr.arb:83]: Forgot this one?

**Review by bdlukaa — APPROVED** [review]
`nServers` and `nEvents` are not translated.

They are like an `if`-clause. When 0, "No events"; when 1, "1 event"; other, "n events".

**Comment by bdlukaa** [comment]
#166 was merged. It added a few localizations. You may need to rebase

File: lib/l10n/app_fr.arb (hunk 1 above)
Comment (sivolc73) [reply lib/l10n/app_fr.arb:83]: Yup, it would be "Dernier rafraîchissement"

File: lib/l10n/app_fr.arb [hunk 2]
```diff
@@ -245,7 +245,7 @@
     }
   },
   "allowAlarms": "Permettre les alarmes",
-  "nextEvents": "Next events",
+  "nextEvents": "Prochains évènements",
   "nEvents": "{n, plural, =0{No events} =1{1 event} other{{n} events}}",
```
Comment (sivolc73) [inline lib/l10n/app_fr.arb:249]: "nEvents": "{n, plural, =0{Aucun évènement} =1{1 évènement} other{{n} évènements}}",

**Comment by sivolc73** [comment]
"nServers": "{n, plural, =0{Aucun serveur} =1{1 serveur} other{{n} serveurs}}",

File: lib/l10n/app_fr.arb (hunk 1 above)
Comment (sivolc73) [reply lib/l10n/app_fr.arb:83]: "lastImageUpdate": "Last Image Update",

**Pull request merged**